src/store/ChartStore.js

```diff
@@ -17,6 +18,7 @@ class ChartStore {
     constructor(mainStore) {
         this.id = ++ChartStore._id_counter;
         this.mainStore = mainStore;
+        this.chartSettingStore = new ChartSettingStore(mainStore);
```

Comment:
@shayanbinary move the chartSettingStore to `/store/index.js` and use `this.mainStore.chartSettingStore` to access it.

Lets follow our convention :-)